app/src/main/AndroidManifest.xml

```diff
@@ -109,7 +109,6 @@
         <activity android:name=".sensors.SensorSHT21" />
         <activity android:name=".sensors.SensorMPU6050" />
         <activity android:name=".sensors.SensorTSL2561" />
-        <activity android:name=".activity.SensorGraphViewActivity" />
```

Comment:
Not a part of this issue. Do clean ups in a separate one.

Reply:
Actually that activity used some instances of OSM , and since osm dependencies are in fdroid version this activity gives errors. That's why I had to remove it.